app/controllers/admin/members_controller.rb

```diff
@@ -24,6 +18,7 @@ def new
 
     def create
       @member = Member.new(member_params)
+      @member.user.terms_of_service = true
```

Comment:
This is to be treated carefully by admin users. An admin user should get the user's consent before creating an account for them.